.github/workflows/tests.yml

```diff
@@ -29,5 +29,5 @@ jobs:
 
     - name: Run Tests
       run: |
-        docker run --rm -v $PWD:/app --network openruntimes-runtimes -w /app phpswoole/swoole:4.8.12-php8.0-alpine sh -c \
+        docker run --rm -v $PWD:/app --network runtimes-1 -w /app phpswoole/swoole:4.8.12-php8.0-alpine sh -c \
```

Comment:
Not critical, Is it worth changing to `phpswoole/swoole:5.1.2-php8.3-alpine` to match the executor PHP version